dist/tools/generate_c11_atomics_cpp_compat_header/generate_c11_atomics_cpp_compat_header.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/sh
```

Comment:
It would help to have usage information here. Alternatively, the could be printed out if the script is called without any argument.

Reply:
I updated the script to print the usage when `-h` or `--help` is passed. It also points to the README.md for documentation.

Additionally, I added a check if no C compiler is specified (either as argument or as environment variable) and to fail than.